deploy/helm/edot-collector/kube-stack/values.yaml

```
@@ -841,75 +480,418 @@ collectors:
+                  default_value: service_transaction
+                - key: elasticsearch.mapping.hints
+                  default_value: [aggregate_metric_double]
+              conditions:
+                - attributes["event.outcome"] != nil and attributes["event.outcome"] == "success"
+              unit: us
+              histogram:
+                buckets: [1]
+                count: Int(AdjustedCount())
+                value: Int(AdjustedCount())
+            - name: event.success_count
+              description: Success count as a metric for service transaction
+              include_resource_attributes:
+                - key: service.name
+                - key: deployment.environment # service.environment
+                - key: telemetry.sdk.language # service.language.name
+                - key: agent.name # set via elastictraceprocessor
+              attributes:
+                - key: transaction.root
+                - key: transaction.type
+                - key: metricset.name
+                  default_value: service_transaction
+                - key: elasticsearch.mapping.hints
+                  default_value: [aggregate_metric_double]
+              conditions:
+                - attributes["event.outcome"] != nil and attributes["event.outcome"] != "success"
+              unit: us
+              histogram:
+                buckets: [0]
+                count: Int(AdjustedCount())
+                value: Double(0)
+      receivers:
+        otlp:
+          protocols:
+            grpc:
+              endpoint: ${env:MY_POD_IP}:4317
+            http:
+              endpoint: ${env:MY_POD_IP}:4318
+      processors:
+        # [Elastic Infra Metrics Processor](https://github.com/elastic/opentelemetry-collector-components/tree/main/processor/elasticinframetricsprocessor)
+        elasticinframetrics:
+          add_system_metrics: true
+          add_k8s_metrics: true
+          drop_original: true
+        # [Attributes Processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/attributesprocessor)
+        attributes/dataset:
+          actions:
+            - key: event.dataset
+              from_attribute: data_stream.dataset
+              action: upsert
+        resource/process:
+          attributes:
+            - key: process.executable.name
+              action: delete
+            - key: process.executable.path
+              action: delete
+        batch:
+          send_batch_size: 1000
+          timeout: 1s
+          send_batch_max_size: 1500
+        # [Elastic Trace Processor](https://github.com/elastic/opentelemetry-collector-components/tree/main/processor/elastictraceprocessor)
+        elastictrace: {} # The processor enriches traces with elastic specific requirements.
+        # [LSM Interval Processor](https://github.com/elastic/opentelemetry-collector-components/tree/main/processor/lsmintervalprocessor)
+        lsminterval:
```

```suggestion
        batch/aggs:
          send_batch_size: 16384 # 2x the default
          timeout: 10s
        lsminterval:
```

Adds batching for lsm interval processor as per https://github.com/elastic/opentelemetry-collector-components/issues/211

We would also need to add the `batch/aggs` processor in the `metrics/aggregated-otel-metrics` pipeline. Note that it already seems to have a batch processor but that seems to be too small for aggregation purposes.

---

Thanks @lahsivjar , pushed these changes in https://github.com/elastic/elastic-agent/pull/6444/commits/af831c8a1d922ce3f22103cfbbfce96fee07b9e1